src/corva/models/scheduled.py

```diff
@@ -67,16 +52,22 @@ def set_timestamp(cls, v: int) -> int:
 
         return v
 
+    @pydantic.root_validator(pre=False, skip_on_failure=True)
+    def set_time_from(cls, values: dict) -> dict:
+        """Calculates time_from field."""
+
+        values["time_from"] = int(values["schedule_start"] - values["interval"] + 1)
```

Comment:
Why `+1`?

Reply:
Because `time_from` means _the first not processed time._ https://github.com/corva-ai/python-sdk/blob/94b2d697609db9a77e1a54d9c51efbe51d2874dd/src/corva/models/scheduled.py#L16

Reply:
Does the users understand this? This must be synced with Node SDK.

Reply:
This field will be easier for users to understand. Compare following exactly the same examples:

Old SDK stream app usage. Pay attention to contents of `query["timestamp"]`.

```python3
import pydantic


class StreamEvent(pydantic.BaseModel):
    asset_id: int
    schedule_start: int
    interval: int


def stream_app(event: StreamEvent, api, cache):
    raw_records = api.get_dataset(
        provider="",
        dataset="",
        query={
            "asset_id": event.asset_id,
            "timestamp": {
                "$gt": event.schedule_start - event.interval,
                "$lte": event.schedule_start,
            },
        },
        sort={},
        limit=1000,
    )
```

New SDK stream app usage. Pay attention to contents of `query["timestamp"]`.

```python3
import pydantic


class StreamEvent(pydantic.BaseModel):
    asset_id: int
    time_from: int
    time_to: int


def stream_app(event: StreamEvent, api, cache):
    raw_records = api.get_dataset(
        provider="",
        dataset="",
        query={
            "asset_id": event.asset_id,
            "timestamp": {
                "$gte": event.time_from,
                "$lte": event.time_to,
            },
        },
        sort={},
        limit=1000,
    )
```

Reply:
I've renamed the fields and expanded ScheduledEvent docstr [here](https://github.com/corva-ai/python-sdk/pull/35/commits/d9d89486900859544df9e2a8e2deaa3fd79a3c60)